### PR TITLE
Update Github issue templates to use issue forms

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard

### Describe the update

Github issue template files are updated to utilize [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms). This involves using YAML configuration files instead of markdown. This should make issue submissions easier for users and will ensure users provide the required information.
    
Note that issue forms are not supported for pull requests currently. 
    
### Expected changes

This is a zero difference update to model output. It will only impact the look of submitting issues on Github once merged into the `main` branch.

### Related Github Issue(s)

- https://github.com/geoschem/geos-chem/issues/1831
- https://github.com/geoschem/geos-chem/pull/2288
- https://github.com/geoschem/GCClassic/pull/58
- https://github.com/geoschem/GCHP/pull/415

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Added
 - GitHub Action config file `.github/workflows/stale.yml`, which replaces StaleBot
- 
+
 ### Removed
-- GitHub config files `.github/stale.yml` and `.github/no-response.yml` 
+- GitHub config files `.github/stale.yml` and `.github/no-response.yml`
+
+### Changed
+- Converted Github issue templates into issue forms using TAML definition files
```
Comment (yantosca) [inline CHANGELOG.md:16]: Typo `TAML` -> `YAML`

**Review by yantosca — CHANGES_REQUESTED** [review]
Thanks @msulprizio!  There is a typo in the changelog but otherwise it is good to merge.

File: CHANGELOG.md (hunk 1 above)
Comment (msulprizio) [reply CHANGELOG.md]: Thanks for catching this. It's now fixed in https://github.com/geoschem/HEMCO/pull/276/commits/7ef08fbba97cd9c9b99b6fdccdda6553efb4d7d7.

Comment (yantosca) [reply CHANGELOG.md]: Great!  Now good to merge.

**Review by yantosca — APPROVED** [review]

**Pull request closed without merging**